stars-core/src/test/kotlin/tools/aqua/stars/core/tsc/TSCMonitorTest.kt

```diff
@@ -46,6 +46,8 @@ class TSCMonitorTest {
               monitor("RootMonitorFalse") { _ -> false } // Always trigger
               monitor("RootMonitorTrue") { _ -> true } // Never trigger
             }
+
+            leaf("leaf") {} // Needed since empty TSC gets skipped by PreEvaluationHook
```

Comment:
I think this is not necessary anymore, as the test checks for tsc.toList().count() >= 0 and not >= 1

Reply:
Fixed.